code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2011,8 +2011,8 @@
 	taste_description = "plastic"
 
 /datum/reagent/glitter
-	name = "Generic Glitter"
-	description = "If you can see this description, contact a coder."
+	name = "standard glitter"
+	description = "Standard sparkles that get everywhere."
```

Comment:
Name and description should be "white glitter" thingies rather than saying standard. It's something standard in the code, but it's not contextual ingame.

Reply:
There is currently also white glitter on line 2033, and I think that this one looks more like light pink anyways.